### DISCUSSION TITLE
About magnetic energy

### DISCUSSION THREAD

**Discussion by mahdishp** [epochpic/epoch#183] [Q&A] [answered]
Dear developers,
Hi
I want to get the density of magnetic energy ((b_x)^2)/2μ.
What method should I use?
The sum of the rows and columns of the matrix that any  cell of matrix is multiplied with itself ,sum(sum(bx.^2)) ? 
Or the sum of the rows and columns of the matrix that multiplied with transpose of the matrix ,sum(sum(bx*bx'))?
Thanks

**Comment by keithbennett** [comment, marked as answer]
The former.

**Comment by keithbennett** [comment]
This should have been opened as a q&a discussion. Moving it there.
